packages/tools/devtools/devtools-view/src/components/utility-components/ConceptCellLayout.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import { Label, TableCellLayout } from "@fluentui/react-components";
+import { InfoLabel } from "@fluentui/react-components/unstable";
+
+/**
+ * {@link ConceptCellLayout} input props.
+ */
+export type ConceptCellLayoutProps = React.PropsWithChildren<{
+	/**
+	 * Icon to display to the left of the label.
+	 */
+	icon: React.ReactElement;
+
+	/**
+	 * (Optional) If specified, will display an "info" badge to the right of the label.
+	 * When clicked, a tooltip will be displayed with the provided contents.
+	 */
+	infoTooltipContent?: React.ReactElement | string;
+}>;
+
+/**
+ * Helper component for rendering the contents of a Table Header Cell intended to represent some Fluid or Devtools
+ * concept.
+ *
+ * Displays child content, with an optional content on the leading side and an optional "info" badge on the trailing
+ * side.
+ */
+export function ConceptCellLayout(props: ConceptCellLayoutProps): React.ReactElement {
```

Comment:
No strong feelings here, but I'd probably have gone with `LabelCellLayout`. The `Concept` piece took me a bit to understand.

Reply:
I like that - will update.